rholang/src/main/scala/coop/rchain/rholang/interpreter/SystemProcesses.scala

```diff
@@ -105,6 +107,36 @@ object SystemProcesses {
           } yield ()
       }
 
+      def getErrorMessagePar(str: String) =
+        RevAddress
+          .parse(str)
+          .swap
+          .toOption
+          .fold(Par.defaultInstance)(GString(_))
+
+      def validateRevAddress: (Seq[ListParWithRandomAndPhlos], Int) => F[Unit] = {
+        case (
+            Seq(
+              ListParWithRandomAndPhlos(
+                Seq(IsString("validate"), IsString(address), ack),
+                rand,
+                _
+              )
+            ),
+            sequenceNumber
+            ) =>
+          for {
+            result <- F.delay(getErrorMessagePar(address))
+            produceResult <- space.produce(
+                              ack,
+                              ListParWithRandom(Seq(result), rand),
+                              persist = false,
+                              sequenceNumber
+                            )(UNLIMITED_MATCH_PHLO)
```

Comment:
OMG we need to make system processes aware of the cost account. Not in this PR ofc.